Binary files pandas-2.2.3.orig/.git/index and pandas-2.2.3/.git/index differ
diff -Nru pandas-2.2.3.orig/pandas/tests/indexes/interval/test_interval_tree.py pandas-2.2.3/pandas/tests/indexes/interval/test_interval_tree.py
--- pandas-2.2.3.orig/pandas/tests/indexes/interval/test_interval_tree.py	2024-09-20 12:21:50.000000000 +0000
+++ pandas-2.2.3/pandas/tests/indexes/interval/test_interval_tree.py	2024-10-01 12:32:47.570061987 +0000
@@ -4,7 +4,10 @@
 import pytest
 
 from pandas._libs.interval import IntervalTree
-from pandas.compat import IS64
+from pandas.compat import (
+    IS64,
+    WASM,
+)
 
 import pandas._testing as tm
 
@@ -190,6 +193,7 @@
         expected = (50 + np.iinfo(np.int64).max) / 2
         assert result == expected
 
+    @pytest.mark.xfail(WASM, reason="GH 23440")
     @pytest.mark.parametrize(
         "left, right, expected",
         [
diff -Nru pandas-2.2.3.orig/pandas/tests/indexing/interval/test_interval_new.py pandas-2.2.3/pandas/tests/indexing/interval/test_interval_new.py
--- pandas-2.2.3.orig/pandas/tests/indexing/interval/test_interval_new.py	2024-09-20 12:21:50.000000000 +0000
+++ pandas-2.2.3/pandas/tests/indexing/interval/test_interval_new.py	2024-10-01 12:34:25.510718974 +0000
@@ -3,6 +3,8 @@
 import numpy as np
 import pytest
 
+from pandas.compat import WASM
+
 from pandas import (
     Index,
     Interval,
@@ -208,7 +210,7 @@
         with pytest.raises(KeyError, match=r"\[6\]"):
             obj.loc[[4, 5, 6]]
 
-
+@pytest.mark.xfail(WASM, reason="GH 23440")
 @pytest.mark.parametrize(
     "intervals",
     [
diff -Nru pandas-2.2.3.orig/pandas/tests/indexing/interval/test_interval.py pandas-2.2.3/pandas/tests/indexing/interval/test_interval.py
--- pandas-2.2.3.orig/pandas/tests/indexing/interval/test_interval.py	2024-09-20 12:21:50.000000000 +0000
+++ pandas-2.2.3/pandas/tests/indexing/interval/test_interval.py	2024-10-01 12:33:50.518484555 +0000
@@ -2,6 +2,7 @@
 import pytest
 
 from pandas._libs import index as libindex
+from pandas.compat import WASM
 
 import pandas as pd
 from pandas import (
@@ -35,6 +36,7 @@
         expected = ser.iloc[2:5]
         tm.assert_series_equal(expected, indexer_sl(ser)[ser >= 2])
 
+    @pytest.mark.xfail(WASM, reason="GH 23440")
     @pytest.mark.parametrize("direction", ["increasing", "decreasing"])
     def test_getitem_nonoverlapping_monotonic(self, direction, closed, indexer_sl):
         tpls = [(0, 1), (2, 3), (4, 5)]
